docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
+* **Normal Controller (Value = 2)**: The normal controller continues to run, and will try to drive the motor at 0% throttle. This can cause vibrating or twitching in velocity mode if the PID gains are high.
+
+This parameter can also be changed through the IQ Control Center if you wish to change this without using DroneCAN.
+
+.. _dronecan_support_telemetry_frequency:
+
+Telemetry Frequency
+####################
+.. table::
+
+	+------------------+----------+
+	| **Name**         | **Type** |
+	+------------------+----------+
+	| telem_frequency  | Integer  |
+	+------------------+----------+
+
+This parameter defines the frequency in Hz with which the telemetry messages (:ref:`uavcan.equipment.esc.Status <dronecan_support_esc_status>` 
+and :ref:`uavcan.equipment.device.Temperature <dronecan_support_device_temperature>`) are broadcast by the module. 
+For example, if this value were set to 10, the telemetry would be sent at a rate of 10 Hz. If this parameter is set to 0, no telemetry messages will be sent.
+
+This parameter can also be changed through the IQ Control Center if you wish to change this without using DroneCAN.
+
+Motor Armed
+##############
+.. table::
+
+	+------------------+----------+
+	| **Name**         | **Type** |
+	+------------------+----------+
+	| motor_armed      | Integer  |
+	+------------------+----------+
+
+This parameter is available on modules that support :ref:`manual_advanced_arming`. It can be used to query and control the armed state of the module. See the 
+section on :ref:`user arming commands over DroneCAN <arming_user_command_dronecan>` for more information.
+
+Motor Stowed
+##############
+.. table::
+
+	+------------------+----------+
+	| **Name**         | **Type** |
+	+------------------+----------+
+	| motor_stowed     | Integer  |
+	+------------------+----------+
+
+This parameter is available on modules that support :ref:`manual_stow_position`. It can be used to command and release stow on modules. See the section on
+:ref:`manual stow over DroneCAN <trigger_manual_stow_dronecan>` for more information.
+
+Stow Status
+############
+.. table::
+
+	+------------------+----------+
+	| **Name**         | **Type** |
+	+------------------+----------+
+	| stow_status      | Integer  |
+	+------------------+----------+
+
+This parameter is available on modules that support :ref:`manual_stow_position`. It reports on the current state of the stow feature on the module. See the
+:ref:`stow_status` section for more information.
+
+Stow Result
+############
+.. table::
+
+	+------------------+----------+
+	| **Name**         | **Type** |
+	+------------------+----------+
+	| stow_result      | Integer  |
+	+------------------+----------+
+
+This parameter is available on modules that support :ref:`manual_stow_position`. It reports on how the previous stow attempt ended. See the
+:ref:`stow_result` section for more information.
 
 Flight Controller Integration

```

Comment:
This links to the PDF on the website, should this change to link to the integration page on read-the-docs instead now?